shared/main-shared.native.js

```diff
@@ -73,7 +73,7 @@ class Main extends Component<void, any, void> {
     // TODO: move Push prompt into route
     const {showPushPrompt, mountPush} = this.props
     return (
-      <Box style={{flex: 1}}>
+      <Box style={{flex: 1, width: '100%'}}>
```

Comment:
its not strictly necessary but the flex 1 doesn't actually fill the entire parent for some reason, this makes it do so (could see in the debugger)